server/models/Collective.js

```diff
@@ -553,8 +554,11 @@ const Collective = sequelize.define(
 
     tags: {
       type: DataTypes.ARRAY(DataTypes.STRING),
+      validate: {
+        validateTags,
+      },
       set(tags) {
-        this.setDataValue('tags', cleanTags(tags));
+        this.setDataValue('tags', sanitizeTags(tags));
```

Comment:
```suggestion
        validateTags(tags);
        this.setDataValue('tags', sanitizeTags(tags));
```

Reply:
@Betree Maybe this is a good middle way?

Technically this would only run when setting the tags array. However, it would run when saving the Edit Collective info page, since that query is currently setting all values regardless whether they change or not.